api/v1/v1.go

```diff
@@ -49,6 +50,8 @@ type PushConfig struct {
 	Registry string
 	// UpdateChanged tells us if we will re-push (update/overwrite) images having same tag, but different digest
 	UpdateChanged bool
+	// PathSeperator defines which path seperator to use (default: "/")
```

Comment:
`PathSeperator` ? 🤔 Maybe `PathSeparator` ?

Reply:
Changed it to `PathSeparator`